README.md

```diff
@@ -215,6 +215,40 @@ app.get('/.well-known/acme-challenge/:acmeToken', function(req, res, next) {
 });
 ```
 
+### PHP Apps
+
+Add the following to `.well-known/acme-challenge/index.php`
+
+```php
+<?php
+$request = $_SERVER['REQUEST_URI'];
+if(preg_match('#^/.well-known/acme-challenge/#', $request) === 0) {
+    return;
+}
+
+if(isset($_ENV['ACME_KEY'])) die(trim($_ENV['ACME_KEY']));
```

Comment:
@dmathieu Should 229 be changed to 

`if(isset($_ENV['ACME_KEY']) && isset($_ENV['ACME_TOKEN']))`

This seems to match the Ruby example then.

`if ENV['ACME_KEY'] && ENV['ACME_TOKEN']`